graphql/resolvers.go

```diff
@@ -1819,6 +1819,33 @@ func (r *mutationResolver) UnschedulePatchTasks(ctx context.Context, patchID str
 	return &patchID, nil
 }
 
+// We only allow scheduling undispatched base tasks for tasks that are failing on the current patch
+func (r *mutationResolver) ScheduleUndispatchedBaseTasks(ctx context.Context, patchID string) ([]*restModel.APITask, error) {
+	opts := data.TaskFilterOptions{
+		Statuses:     evergreen.TaskFailureStatuses,
+		BaseStatuses: []string{evergreen.TaskUndispatched},
+	}
+	tasks, _, err := r.sc.FindTasksByVersion(patchID, opts)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Could not fetch tasks for patch :%s ", err.Error()))
```

Comment:
nit: "patch: %s"

Reply:
oh and can we maybe also add ' ', like "patch: '%s'"?